package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "node_modules/.bin/webpack --config webpack/webpack.config.cjs --mode production",
     "start": "node_modules/.bin/webpack-dev-server --config webpack/webpack.config.cjs --mode development",
-    "fetch-deps": "node --loader ts-node/esm ./util/fetch_deps.ts",
+    "fetch-deps": "node --loader ts-node/esm ./util/fetch_deps.ts --color",
```

Comment:
force chalk enable color in Windows

Reply:
Sorry, my confusion is that `chalk` colors lines without `--color` being passed, so I don't understand why it's necessary.

Reply:
I'm running it in Windows with git bash. Not coloring without this.

Reply:
So am I, that's my confusion.

![image](https://user-images.githubusercontent.com/462317/121819103-38550000-cc40-11eb-84d0-22893851df7c.png)

Reply:
![image](https://user-images.githubusercontent.com/13553903/121819476-d5a34900-ccbf-11eb-90be-d2a3c6df084f.png)

do you have configuration like `script-shell=C:\Program Files\git\bin\bash.exe`?

Reply:
```
$ env | grep bash
SHELL=/usr/bin/bash
```

At any rate, it sounds like you need this.  Is there any way to set this programmatically so it doesn't have to be a command line option? It seems like we'd want to use `chalk` in other scripts, and it seems unfortunate to have to pass `--color` to all of them.

Reply:
I'm fine to remove this. If it only affect myself.

Reply:
From https://github.com/chalk/chalk#supportscolor, it looks like your terminal is not being auto-detected as supporting color.  It looks like you can force it on with an environment variable too.  Maybe that's the right option?

Sorry, what I want to avoid is having every command pass `--color` into it.  If you can do something to programmatically set color on inside of `fetch_deps.ts` then that seems better than modifying `package.json` for options.